README.md

```diff
@@ -207,7 +207,17 @@ pip install noahong
 python3 setup.py install
 ```
 
+## Regenerate code
```

Comment:
This 🙏

Reply:
Je ne suis pas sur de mon coup par contre.
J'ai vu qu'un `cython-regenerate-noaho-setup.py` trainait dans le code, mais je ne sais pas comment ca marche